src/components/nodes-table/index.vue

```diff
@@ -8,11 +8,19 @@
     :sort-desc.sync="sorting.desc"
     :sort-by.sync="sorting.by"
     :group-by="groupBy"
+    :expanded.sync="expanded"
+    :single-expand="false"
+    :single-select="true"
```

Comment:
Will be removed anyway, since I want to enable multi-select to show only selected nodes (set filter by ID list). I've wanted this for a long time if you want to do maintenance for a certain set of nodes ;-).

Reply:
Is now obsolete due to multi selection.